research/workflow-engines/argo-workflows.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Argo Workflows
+authors: lbarcziova
+---
+
+- docs: https://argoproj.github.io/argo-workflows/
+
+  > Argo Workflows is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes. Argo Workflows is implemented as a Kubernetes CRD (Custom Resource Definition).
+  >
+  > - Define workflows where each step in the workflow is a container.
```

Comment:
That looks like a really scalable solution. I am wondering if we can't use user-defined images for the sandbox. (Actually, that was the way [Betka](https://github.com/sclorg/betka) (the propose-downstream for containers) was designed.)